lib/fluent/plugin/out_logdna.rb

```diff
@@ -84,7 +84,8 @@ def send_request(body)
       url = "/logs/ingest?hostname=#{@host}&mac=#{@mac}&ip=#{@ip}&now=#{now}&tags=#{@tags}"
       @ingester.headers('apikey' => @api_key,
                         'content-type' => 'application/json')
-               .post(url, json: body)
+                .timeout(connect: 10, write: 10, read: 10)
```

Comment:
Can this be configurable? + why 10 seconds?

Reply:
Make it configurable and chose a default to match the Python library (we're not consistent between libraries here).